src/abis/nithin_Token.js

```diff
@@ -0,0 +1,357 @@
+export const nithinTokenABI = [
+  {
+    inputs: [],
```

Comment:
rename this abi according to the existing file. you may save the existing abi as _bkp.